apps/native-component-list/src/screens/DocumentPickerScreen.tsx

```diff
@@ -10,14 +10,21 @@ export default function DocumentPickerScreen() {
   const [document, setDocument] = React.useState<DocumentPicker.DocumentResult | null>(null);
 
   const openPicker = async () => {
+    const time = Date.now();
     const result = await DocumentPicker.getDocumentAsync({
       copyToCacheDirectory: copyToCache,
     });
+    console.log(`Duration: ${Date.now() - time}ms`);
+    console.log(`Results:`, result);
```

Comment:
Do we want to leak such `console.log`s to production apps? 🤔 
Maybe we can hide them behind some `DEBUG=1` env variable or so? 🤔

Reply:
shouldn't matter since it's just in native-component-list